types/src/validator_verifier.rs

```diff
@@ -192,10 +192,11 @@ impl<PublicKey: VerifyingKey> ValidatorVerifier<PublicKey> {
     where
         T: Into<PublicKey::SignatureMaterial> + Clone,
     {
-        for author in aggregated_signature.keys() {
-            if self.author_to_public_keys.get(&author) == None {
-                return Err(VerifyError::UnknownAuthor);
-            }
+        if aggregated_signature
```

Comment:
What is the motivation for this type of conversion? From a readability perspective, both of them seem equally accessible.

Reply:
Thanks for the pushback, uploading a more readable version.